package.json

```diff
@@ -37,6 +37,7 @@
     "codecov.io": "^0.1.6",
     "core-js": "^2.4.1",
     "express": "^4.16.3",
+    "hoek": "^5.0.3",
```

Comment:
Why did you add this dependency additionally ?

Reply:
I updated the hoek dependency using `npm` and everything was done by dependency manager npm only.

Reply:
What command did you use ?